### DISCUSSION TITLE
two interfaces

### DISCUSSION THREAD

**Discussion by korovkintaptap** [pivpn/pivpn#1514] [Support]
My Raspberry pi has to wired connections (the built in ethernet connection on the board and a USB to Ethernet adapter) 

One is connected to a cable ISP (slow upload speed) 
and the other is connected a fiber ISP 

i would like to have a choice which WG VPN to establish dynamically when i create the profile 

one that uses the first ISP end to end (for both incoming and outgoing connections) 
and the other profile that uses the second ISP for everything (incoming and outgoing connections)

as far as i see, PiVPN can only have one output device at the moment, or am i missing something? 

Thank you

**Comment by korovkintaptap** [comment]
the two interfaces: 

```
enx54af975c047e: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.89.110  netmask 255.255.255.0  broadcast 192.168.89.255
        inet6 fe80::767d:90e0:6a17:512d  prefixlen 64  scopeid 0x20<link>
        inet6 fdb4:d607:ab4a:0:b2eb:a69a:2605:656  prefixlen 64  scopeid 0x0<global>
        ether 54:af:97:5c:04:7e  txqueuelen 1000  (Ethernet)
        RX packets 379136  bytes 229047938 (218.4 MiB)
        RX errors 0  dropped 2  overruns 0  frame 0
        TX packets 514183  bytes 668616460 (637.6 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

enxb827eb4ddb7d: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.86.202  netmask 255.255.255.0  broadcast 192.168.86.255
        inet6 fe80::474b:1b68:fee5:3f38  prefixlen 64  scopeid 0x20<link>
        ether b8:27:eb:4d:db:7d  txqueuelen 1000  (Ethernet)
        RX packets 5874699  bytes 2072126635 (1.9 GiB)
        RX errors 10707  dropped 361  overruns 0  frame 22
        TX packets 3462742  bytes 995482399 (949.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

**Comment by orazioedoardo** [comment]
> as far as i see, PiVPN can only have one output device at the moment, or am i missing something?

Correct.

> i would like to have a choice which WG VPN to establish dynamically when i create the profile

It is possible but I think this would be out of scope for pivpn.
